detox/src/devices/Device.js

```diff
@@ -140,6 +140,8 @@ class Device {
   }
 
   async setLocation(lat, lon) {
+    lat = (lat + '').replace('.', ',');
+    lon = (lon + '').replace('.', ',');
```

Comment:
Thanks for the contribution!
If possible, please use `String()` as @isnifer's suggests. We will merge it ASAP.